eruditorg/templates/public/journal/partials/issue_detail_section.html

```diff
@@ -1,66 +1,17 @@
-  {% if section1.titles.main %}
-  <h3 class="section">
-    {{ section1.titles.main }}
-    {% for paral in section1.titles.paral %}/ {{ paral }}{% endfor %}
-  </h3>
+{% for group in articles_per_section.groups %}
+  {% if group.type == 'objects' %}
+    <ol class="issue_detail--articles unstyled section-{{ group.level }}">
+      {% for article in group.objects %}
+      {% include "public/journal/partials/issue_detail_article_item.html" %}
+      {% endfor %}
+    </ol>
+  {% else %}
+      <h{{ group.level|add:"1" }} class="section">
+      {{ group.titles.main }}
```

Comment:
Also, make sure to display all languages in the order that is presented in summary.xml 

See: http://localhost:8000/fr/revues/mi/2015-v19-n2-mi01857/ 
Also: http://retro.erudit.org//revue/mi/2015/v19/n2/index.xml

The section titles, in this case, should be displayed in the following order: FR / EN / ES (right now, they're displayed as FR / ES / EN) 

```
<surtitre>
Dossier spécial : Entreprendre et innover dans une économie globalisée...
</surtitre>
<surtitreparal lang="en">
Special Edition: Undertaking and innovating in a globalised economy...
</surtitreparal>
<surtitreparal lang="es">
Dossier especial: Emprender e innovar en una economía globalizada…
</surtitreparal>
```